lib/rules/require-shorthand-arrow-functions.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Require shorthand arrow function expressions when returning a single statement.
```

Comment:
No I think you're right about it being confusing. I'm not sure what the best wording should be. I think it's since arrow functions are actually the shorthand of normal functions anyway.

One distinction is that one way is an expression body and the one with braces is a statement body (block)

```js
// Expression bodies
var odds = evens.map(v => v + 1);
// Statement bodies
nums.forEach(v => {
  if (v % 5 === 0)
    fives.push(v);
});
```

Maybe even the rule name should be changed but I'm not sure.